docs/modules/mysql.md

```diff
@@ -1,9 +1,10 @@
 # MySQL
 
 ### Purpose
+[MySQL](https://www.mysql.com/) is a database management system. MySQL will organize and provide access to databases where sites can store information.
 
 ### Dependencies
-
+MySQL can be run on Windows, OSX, RHEL/CentOS, Ubuntu, and Fedora. MySQL will require PHP5. These packages can also be downloaded from APT by running `sudo apt-get install mysql-server php5-mysql`. Once the packages are installed we need to create a database directory structure by typing `sudo mysql_install_db`. For more information, please refer to the [documentation](https://dev.mysql.com/doc/). 
```

Comment:
I'm not sure that the dependency section is the best place for detailed installation instructions. If we were to follow the same pattern for the Wordpress or Magento modules, would we be expecting to document complete top-to-bottom installation directions? I feel that the dependency section should specifically be for mentioning any prerequisites that need to be in place on the system before the module can be installed. Installation instructions are outside of the scope of what we're trying to document here, and again, singling out one or two operating systems doesn't fit with the theme of giving a broad look at the module.